draft-wang-ppm-differential-privacy.md

```diff
@@ -239,11 +254,133 @@ The following considerations are out-of-scope for this document:
+make `EPSILON` so close to `0` that the difference between `p(S, D1, R)` and
+`p(S, D2, R)` is negligible. However, this requires adding more noise, which
+has an adverse impact on utility. Most applications will accept a
+non-negligible bound in order to achieve reasonable utility. See
+{{dp-explainer}} for details.
+
+This notion of `EPSILON`-DP is sometimes referred to as "pure-DP". The
+following is a relaxation of pure-DP, called "approximate-DP", from {{DR14}}. A
+randomized algorithm `R` is called "`(EPSILON, DELTA)`-DP" if for all
+neighboring `D1` and `D2` and all possible aggregate results `S`, the following
+inequality holds:
+
+~~~
+p(S, D1, R) <= exp(EPSILON) * p(S, D2, R) + DELTA
+~~~
 
-TODO
+Compared to pure-DP, approximate-DP loses an additive factor of `DELTA` in the
+bound. This is to account for certain DP mechanisms that have some desirable
+properties, but are not pure-DP. See {{mechanisms}} for details.
+
+Other variants of DP are possible; see the literature review in
+{{dp-explainer}} for details.
+
+## Trust Models
+
+When considering whether a given DP policy is sufficient for DAP, it is not
+enough to consider the mechanisms used in isolation. It is also necessary to
+consider the process by which the policy is executed. In particular, our threat
+model for DAP considers an attacker that participates in the upload,
+aggregation, and collection protocols and may use its resources to attack the
+underlying primitives (VDAF {{!VDAF}}, HPKE {{!RFC9180}}, and the transport
+security protocol).
+
+To account for such attackers, our goal for DAP is "computational-DP" as
+described by {{MPRV09}}, which formalizes the amount of information an active
+attacker can glean about the measurements over the course of its attack on the
+protocol. We consider an attacker that controls the network and statically
+corrupts a subset of the parties.
+
+We say the protocol is pure-DP (resp. approximate-DP) if the view of any
+computationally bounded attacker can be efficiently simulated by a simulator
+that itself is pure-DP (or approximate-DP) as defined above. (The simulator
+takes the measurements as input and outputs a value that is computationally
+indistinguishable from the transcript of the protocol's execution in the
+presence of the attacker.)
+
+Whether this property holds for a given DP policy depends on which parties can
+be trusted to execute the protocol correctly (i.e., which parties were not
+corrupted by the attacker). The DP policies described in this document are
+secure in one of two trust models described below.
+
+### OAMC: One-Aggregator-most-Clients
+
+Assume that most Clients and one Aggregator is honest. The other
+Aggregator and the Collector are controlled by the atacker. The degree of
+privacy provided by the DP policy (i.e., `EPSILON`) degrades as the number of
+honest Clients decreases.
+
+### OA: One-Aggregator
+
+Assume that at least one Aggregator is honest. The other Aggregator(s), the
```

Comment:
perhaps enough to just say Aggregator, without (s), as other parts of the PR does

Reply:
yes, good catch!